baseplate/context/memcache/__init__.py

```diff
@@ -5,11 +5,12 @@
 
 from pymemcache.client.base import PooledClient
 
-from . import ContextFactory
-from .. import config
+from .lib import make_memcache_serializer, memcache_deserializer
+from baseplate.context import ContextFactory
+from baseplate import config
 
 
-def pool_from_config(app_config, prefix="memcache.", **kwargs):
+def pool_from_config(app_config, prefix="memcache."):
```

Comment:
probably also good to note somewhere in the module docs that we implement serde for you.